mongodb/src/main/java/com/yahoo/ycsb/db/OptionsSupport.java

```diff
@@ -76,7 +76,7 @@ else if ("acknowledged".equals(writeConcernType)) {
                 result = addUrlOption(result, "w", "1");
             }
             else if ("journaled".equals(writeConcernType)) {
-                result = addUrlOption(result, "journal", "true");
+                result = addUrlOption(result, "j", "true");
```

Comment:
The documentation for the standard URI says to use "journal":

  http://docs.mongodb.org/manual/reference/connection-string/#uri.journal

Does that not work with the Java Driver?